### PR TITLE
Add docs for the show command and backpressure_strategy options

### PR DESCRIPTION
**Pull request by adriansr**
Document the following features of Auditbeat's auditd module:

* `backpressure_strategy` option.
* `show auditd-status` and `show auditd-rules`.

### CODE REVIEW

**Comment by andrewkroh** [comment]
On a related note, I think it could be potentially confusing for users to show rules in multiple places. I think it would benefit us to steer users to one location and highlight it in our documentation.

I'm thinking to update the documentation examples to show the contents the .conf file rather than the rules in the yml file. And remove the commented out rules under `audit_rules` in the provided config file.

I assume the getting started guide would need similar updates and also mention removing the `.disabled` file extension.

**Comment by adriansr** [comment]
Thanks @dedemorton , have another look now

File: auditbeat/docs/modules/auditd.asciidoc [hunk 1]
```diff
@@ -146,6 +196,28 @@ loaded after the rules declared in `audit_rules` are loaded. Wildcards are
...
+The possible values are:
+
+- `auto` (default): {beatname_uc} uses the `kernel` strategy, if supported, or
+falls back to the `userspace` strategy.
+- `kernel`: {beatname_uc} sets the `backlog_wait_time` in the kernel's
+audit framework to 0. This causes events to be discarded in the kernel if
+the audit backlog queue fills to capacity. Requires a 3.14 kernel or
+newer.
+- `userspace`: {beatname_uc} drops events when there is backpressure
+from the publishing pipeline. If no rate_limit is set, {beatname_uc} sets a rate
```
[the remark this answers is not in the document]
Comment (adriansr) [reply auditbeat/docs/modules/auditd.asciidoc]: done

**Comment by adriansr** [comment]
jenkins, test this please

**Pull request merged**